stable/nats/templates/statefulset.yaml

```diff
@@ -0,0 +1,117 @@
+apiVersion: apps/v1
```

Comment:
use `apps/v1beta2`, the `apps/v1` is pretty new and does not work unless you have the latest k8s versions

Reply:
👍